CHANGELOG.md

```diff
@@ -17,6 +17,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 - Ability to reverse search in ``cartridge enter`` and ``cartridge connect`` commands.
 
+### Changed
+
+- Updated Go to version 1.17
```

Comment:
I would clarify this entry. Updated in CI, right? Whether is works on 1.17 before? Or those go.mod/sum changes were required?

Reply:
Fixed